apigee-test-shared/README.md

```diff
@@ -0,0 +1,25 @@
+# Apigee-test-shared
+Docker image used to run unit tests on Apigee proxies
+
+## Mounting tests
+To use this image your tests should be mounted into /code/src using this folder structure
+```
+src  
+.
+.
+.
+└───<insert code to be tested here>
+│   
+└───tests
+    |
+    └───<insert tests here>
+```
+
+## Test filename naming convention
+
+All units test filenames should be named: \<file_under_test\>.test.js 
+
+## Building the container
+```
+docker run --rm -it -v ${PWD}/src/tests:/code/src/codeToTest -v ${PWD}/src/gateway:/code/src/codeToTest test-shanes2:latest  /bin/bash -ci  './bin/test.sh'
+```
```

Comment:
🚲 nitpick: wide lines like this will be hard to read in the docs, if they introduce horizontal scrolling. Breaking it up onto multiple lines also gives you a way to group related arguments visually.  IMO, this is nicer:

~~~markdown
```
docker run --rm --it \
    -v ${PWD}/src/tests:/code/src/codeToTest \
    -v ${PWD}/src/gateway:/code/src/codeToTest \
    test-shanes2:latest  \
    /bin/bash -ci  './bin/test.sh'
```
~~~

Reply:
Why do we `/bin/bash -ci './bin/test.sh'`? 

Wouldn't `./bin/test.sh` suffice?

Reply:
🚲 Alternatively, we could add `CMD ./bin/test.sh` to the Dockerfile and not have to specify what to run here.

Reply:
> 🚲 Alternatively, we could add `CMD ./bin/test.sh` to the Dockerfile and not have to specify what to run here.

This idea is that this image is shared across different repos as a base image. Each one will mount something different and then at that point we want to run the test script.

> Why do we `/bin/bash -ci './bin/test.sh'`?
> 
> Wouldn't `./bin/test.sh` suffice?

I'll try it!